File: yep/src/main/java/catchla/yep/adapter/iface/ItemClickListener.java
package catchla.yep.adapter.iface;

import android.support.v7.widget.RecyclerView;

import catchla.yep.view.holder.TopicViewHolder;

/**
 * Created by mariotaku on 15/5/29.
 */
public interface ItemClickListener {

    void onItemClick(int position, RecyclerView.ViewHolder holder);

}

File: src/main/java/catchla/yep/adapter/iface/ItemClickListener.kt
package catchla.yep.adapter.iface

import android.support.v7.widget.RecyclerView

import catchla.yep.model.Skill
import catchla.yep.view.holder.FriendGridViewHolder
import catchla.yep.view.holder.TopicViewHolder

/**
 * Created by mariotaku on 15/5/29.
 */
interface ItemClickListener {

    fun onItemClick(position: Int, holder: RecyclerView.ViewHolder)

}
